components/Tabs/src/TabBarView/MDCTabBarView.m

```diff
@@ -457,6 +547,95 @@ - (CGSize)expectedSizeForView:(UIView *)view {
   return expectedItemSize;
 }
 
+/// Sets _selectionIndicator's bounds and center to display under the item at the given index with
+/// no animation. May be called from an animation block to animate the transition.
+- (void)updateSelectionIndicatorToIndex:(NSUInteger)index {
+  if (index == NSNotFound || index >= self.items.count) {
+    // Hide selection indicator.
+    self.selectionIndicatorView.bounds = CGRectZero;
```

Comment:
Maybe use the hidden property to hide the indicator view?

Reply:
This is the current logic - I believe it allows animation of the indicator as a "shrinking" effect rather than a fade-out. In either case, it hasn't caused bugs to be filed yet, so I'd prefer to keep it as-is and [the selection indicator animation/layout code can be cleaned-up in the future](https://github.com/material-components/material-components-ios/issues/7748).